### PR TITLE
Add cookie name filter

### PR DESCRIPTION
**Pull request by beccahughes**
This filter will be used for more specific cookie invalidation.

***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on May 18, 2020, 10:39 PM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Frawcdn.githack.com%2FWICG%2Fmedia-feeds%2F2b04e45c0d1c0125edb7e094719f6571c524dd62%2Findex.html%3FisPreview%3Dtrue)

```
[33m📡 HTTP Error 522:[39m [36mhttps://rawcdn.githack.com/WICG/media-feeds/2b04e45c0d1c0125edb7e094719f6571c524dd62/index.html[39m
```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20WICG/media-feeds%2327.)._
</details>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -97,7 +97,12 @@ <h2>Security and Privacy Considerations</h2>
           The user visits the website or an <a>associated origin</a>.
         </li>
         <li>
-          The cookies expire or are deleted on the website or an <a>associated origin</a>.
+          If there is a <a>cookie name filter</a> that matches the name of the cookie
```
Comment (sgbowen) [inline index.html:100]: Can you reword this to specify the cookie must be on the website or associated origin?

This is hard to cram into a sentence, but maybe something like:
"If there is a cookie name filter that matches the name of a cookie on the website or associated origin and that cookie's value changed."

File: index.html [hunk 2]
```diff
@@ -97,7 +97,12 @@ <h2>Security and Privacy Considerations</h2>
...
+          that changed.
+        </li>
+        <li>
+          If there is no <a>cookie name filter</a>. Then when any cookie expires or is
```
Comment (sgbowen) [inline index.html:104]: nit: This can be one sentence. "If there is no cookie name filter, then when any cookie..."

File: index.html [hunk 3]
```diff
@@ -113,6 +118,14 @@ <h2>Security and Privacy Considerations</h2>
       <a data-cite="URL#concept-url-scheme">scheme</a>.
     </p>
 
+    <p>
+      A <dfn>cookie name filter</dfn> is the name of a cookie that will be used by
```
Comment (sgbowen) [inline index.html:122]: I initially misread this sentence because it sounded like the user agent would invalidate the data by means of the cookie (i.e., editing the cookie somehow). But actually the cookie change just signals the user agent to invalidate using its own mechanism.

I'd suggest something like: "A cookie name filter is the name of a cookie that signals the user agent to invalidate the contents of the feed..."

**Comment by beccahughes** [comment]
Thanks, updated
